frontend/pages/policies/ManagePoliciesPage/ManagePoliciesPage.tsx

```diff
@@ -339,9 +327,14 @@ const ManagePolicyPage = ({
     }
   );
 
+  console.log("teamPoliciesCount", teamPoliciesCount);
+
   const canAddOrDeletePolicy: boolean =
     isGlobalAdmin || isGlobalMaintainer || isTeamMaintainer || isTeamAdmin;
   const canManageAutomations: boolean = isGlobalAdmin || isTeamAdmin;
+  const hasPoliciesToAutomateOrDelete: boolean = teamIdForApi
+    ? !!(!isFetchingTeamCount && teamPoliciesCount && teamPoliciesCount > 0)
+    : !!(globalPoliciesCount && globalPoliciesCount > 0);
```

Comment:
This should do the trick when the API is implemented, I think the buggy UI you're seeing is because your API count is returning 2 in the background